.github/workflows/test-in-app-messaging-prs.yml

```diff
@@ -47,8 +47,14 @@ jobs:
       commit: ${{ github.event.pull_request.head.sha }}
       repository: ${{ github.event.pull_request.head.repo.full_name }}
     secrets:
-      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }} # TODO: remove this
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }} # TODO: remove this
```

Comment:
Awaiting tfn verification until we can fully migrate away from these credentials.

Reply:
nit: do we need to add a reason when to "remove this"? otherwise it would be weird that who made the comment didn't "remove this" 🤣

Reply:
Good point! Let me reword it to:
```
# TODO: migrate away from this account once tfn is migrated
`````

Reply:
Done! 7289fe67b